packages/chatbot-eval-mongodb-public/src/braintrust/evaluateRagConversationsData.ts

```diff
@@ -0,0 +1,74 @@
+import { assertEnvVars, ObjectId } from "mongodb-rag-core";
+import "dotenv/config";
+import { strict as assert } from "assert";
+import {
+  ConversationGeneratedData,
+  makeMongoDbGeneratedDataStore,
+  SomeGeneratedData,
+} from "mongodb-chatbot-evaluation";
+import { evaluateRagConversationsReferenceFree } from "./evaluateRagConversationsReferenceFree";
+
+const runs = [
+  // {
+  //   model: "gpt-35",
+  //   generatedDataRunId: new Object("669019be0aae97ffdafcd0ff").toString(),
+  // },
+  // {
+  //   model: "gpt-4o",
+  //   generatedDataRunId: new Object("66901df714b4b4953729c844").toString(),
+  // },
+  // {
+  //   model: "gpt-4o-mini",
+  //   generatedDataRunId: new Object("66c35ec36e955a072a6f8784").toString(),
+  // },
+  {
+    model: "gpt-35-preprocessor-gpt4o-mini-responder",
+    generatedDataRunId: new Object("66c37e6c97b47998dd59e87e").toString(),
+    description: `Uses GPT-3.5 for pre-processsing tasks: metadata extraction, user input guardrail, and query rewriting.
+Uses GPT-4o mini for the main responder.`,
+  },
+];
+async function main() {
+  const {
+    MONGODB_CONNECTION_URI,
+    MONGODB_DATABASE_NAME,
+    // NOTE: This is using an OpenAI key from OpenAI directly
+    // (not Azure OpenAI).
+    OPENAI_OPENAI_API_KEY,
```

Comment:
To be clear, we have separate OpenAI OpenAI and Azure OpenAI credentials that we can use through Radiant? Will we need to distribute these / generate new ones for folks to run this locally?

Reply:
yes, we'd need to distribute for folks to run locally. i've contributed to an existing issue for autoevals to have first class support for azure openai, so hopefully this problem will go away soon.

https://github.com/braintrustdata/autoevals/issues/89